libs/blocks/adobetv/adobetv.css

```diff
@@ -1,4 +1,5 @@
 @import url('../../styles/iframe.css');
+@import url('../video/video.css');
```

Comment:
What exactly does adobetv use from the video styles? Doesn't that come with its own style rules OOTB?

Reply:
Adobe tv videos which make use of the `decorateAnchorVideo` function have the accessibility controls but fail to load the css required for it.